### DISCUSSION TITLE
Disable logging to the WorkflowExecutionLogRecords table

### DISCUSSION THREAD

**Discussion by maxche91** [elsa-workflows/elsa-core#3402] [Q&A]
Too many records are stored in the [Elsa].[WorkflowExecutionLogRecords] table due to which the database reaches 20 GB,
Is there any way to disable logging to the WorkflowExecutionLogRecords table?

**Comment by Kvariani** [comment]
I think this will be useful function.

@sfmskywalker I can implement this functionality by modifying WorkflowExecutionLog class and add some DI parameter in constructor.

I have never contributed this project, but if you think that this is good idea, I'll do it.

Thanks.

Beka Kvariani

**Reply by maxche91** [reply]
@sfmskywalker , @Kvariani  it would be really useful from a practical point of view
